### DISCUSSION TITLE
Cypress crashes and shows white window with no infos

### DISCUSSION THREAD

**Discussion by thepixelhead** [cypress-io/cypress#28445] [Questions and Help]
At the moment my cypress test crashes and the cypress monitor shows completly white frame. No column for the test steps or browser window visible. The crash happens once in a while, not always but too often and at different test steps, so no stringent behaviour. When crashing there is no message by cypress that says what happened.
Is there a possibility that a cypress test script is too long? My script file is 6000 lines long. Is it too much? Are there a max code lines? Any idea? Thank you very much!

Using Cypress v12.14.0

EDIT:

I just see there is a message in the terminal frame of the editor:

If you're running lots of tests on a memory intense application.
  - Try increasing the CPU/memory on the machine you're running on.
  - Try enabling experimentalMemoryManagement in your config file.
  - Try lowering numTestsKeptInMemory in your config file during 'cypress open'.

**Comment by adamalston** [comment]
Does the 6000-line long file only have one test in it or are there multiple?

If there are multiple tests in this file and you're running the entire thing in the test runner without limiters, your browser may be running out of memory. Can you try setting `numTestsKeptInMemory` to `0` in your configuration file?
